loopdb/loop.go

```diff
@@ -26,6 +27,11 @@ type SwapContract struct {
 	// HTLC.
 	ReceiverKey [33]byte
 
+	// KeyLocator is the key locator (family and index) for the receiver
+	// key if this is a loop out contract, or the sender key if this is a
+	// loop in contract.
+	KeyLocator keychain.KeyLocator
```

Comment:
nit: maybe name this to be remote/server/peer key to help us understand that this is used for key aggregation.